### PR TITLE
Moved TTS setting into GeneralSettingsMenu

### LINKED ISSUE #1509: Enable TTS in Game Options is transient
# Description
Game volume and zoom level are saved between game runs but Enable TTS is not

## Category
BUG

## Detailed Bug Report
**Build**: [d5b5627](https://github.com/unitystation/unitystation/commit/d5b56271e1390db3337adb9171770526b01901d5)
**Unity Version**: 2018.3.4f1Personal
OS: Windows 10 Pro x64

This client preference should be saved so it can be loaded in subsequent runs, saving the player the effort of having to quickly get it enabled again after they've already joined in on a round.

## Steps to Reproduce
Please enter the steps to reproduce the bug or behaviour:
1. Run unitystation
2. Join round
3. Change preferences, enable TTS
4. Log out, quit game
5. Run unitystation
6. Join round
7. Check preferences. TTS will be disabled again

### PR DESCRIPTION
**Pull request by vlandemart**
- Moved TTS setting into GeneralSettingsMenu
- TTS setting is now being saved (PlayerPrefs)

### Purpose
- Saves TTS setting. I've decided not to inflate UIManager anymore and moved logic to GeneralSettingsMenu.
- Fixes #1509 

### Open Questions and Pre-Merge TODOs

- [x]  This fix is tested on the same branch it is PR'ed to.
- [x]  I correctly commented my code
- [x]  My code is indented with tabs and not spaces
- [x]  This PR does not include any unnecessary .meta, .prefab or .unity (scene) changes
- [x]  This PR does not bring up any new compile errors
- [x]  This PR has been tested in editor
- [ ]  This PR has been tested in multiplayer

### Notes:
Currently, it saves using PlayerPrefs and Initializes GeneralSettingsMenu in UIManager 's Start() method

### CODE REVIEW

File: UnityProject/Assets/Scripts/UI/UIManager.cs [hunk 1]
```diff
@@ -107,6 +109,11 @@ public static string SetToolTip
 	/// </summary>
 	public static bool IsOxygen { get; set; }
 
+	private void Start()
+	{
+		generalSettingsMenu.Init();
```
Comment (fomalsd) [inline UnityProject/Assets/Scripts/UI/UIManager.cs:114]:
```
NullReferenceException: Object reference not set to an instance of an object
UIManager.Start () (at Assets/Scripts/UI/UIManager.cs:114)
```
It's not assigned

**Review by fomalsd — CHANGES_REQUESTED** [review]
generalSettingsMenu is not assigned and it produces NRE on game start from Lobby (see other comment)

File: UnityProject/Assets/Scripts/UI/UIManager.cs (hunk 1 above)
Comment (vlandemart) [reply UnityProject/Assets/Scripts/UI/UIManager.cs:114]: Damn, must've forgotten to apply prefab. Sorry, should have double checked if I missed anything.

Fixed it.

**Comment by DooblyNoobly** [comment]
Heads up @vlandemart, it's best to make a new branch for any fix / feature on your fork so you don't mess up your develop branch for when you need to pull latest from the origin repo. Keep it as clean as possible or else you'll find yourself having to refork

**Pull request merged**